### PR TITLE
WIP, do not merge yet, get calendar viewtype in working shape

### PR DESCRIPTION
**Pull request by yahelnachum**

### CODE REVIEW

File: src/components/Introduction.jsx [hunk 1]
```diff
@@ -328,7 +340,8 @@ const Introduction = () => {
               style={styles.button}
               onClick={() => {
                 if (
-                  treatment.viewType === ViewType.word &&
+                  (treatment.viewType === ViewType.word ||
+                    treatment.viewType === ViewType.calendarWord) &&
```
Comment (yahelnachum) [inline src/components/Introduction.jsx:344]: this change might not be needed, re-review later

Comment (pcordone) [reply src/components/Introduction.jsx:344]: That is, it is never clicked.  I can change it to be similar to the useEffect code that enables the button.

Comment (pcordone) [reply src/components/Introduction.jsx:344]: I fixed the logic on main.  I removed the check if it is a word treatment. That isn't needed.  Be sure to merge my changes in from main.  I did a lot of little changes.

**Comment by yahelnachum** [comment]
not applicable anymore, closing for now

**Pull request closed without merging**